core/src/main/java/hudson/model/Job.java

```diff
@@ -868,7 +868,7 @@ public RunT getNearestBuild(int n) {
     }
 
     /**
-     * Gets the latest build #m that satisfies {@code m&lt;=n}.
+     * Gets the newest build #m that satisfies {@code m ≤ n}.
```

Comment:
Arguably clearer phrasing. Encountered while studying the behavior of `getNearestOldBuild(0)` (see elsewhere).